WebRoot/js/zimbraMail/core/ZmAppCtxt.js

```diff
@@ -2328,3 +2328,25 @@ ZmAppCtxt.prototype.showError = function(params) {
     dlg.setMessage(errMsg, params.details, params.style || DwtMessageDialog.WARNING_STYLE, params.title);
     dlg.popup(null, params.noReport !== false);
 };
+
+/**
+ * Get sharing folders
+ *
+ * @param {Object} folder     current node
+ * @param {string} type       folder type
+ * @param {Array}  result     an array of sharing folders
+ */
+ZmAppCtxt.prototype.getSharingFolders =
```

Comment:
I think getSharedFolders would me more meaningful. what you say?

Reply:
I didn't come up with any good name... How about this?
* findSharingFolders or get(find)SharingFolderObjects

Reply:
as per https://github.com/Zimbra/zm-web-client/pull/666#issuecomment-1021080072, let's keep getSharedFolders.

Reply:
changed to `getSharedFolders`